tests/baselines/reference/complicatedPrivacy.types

```diff
@@ -61,7 +61,7 @@ module m1 {
     }
 
     export function f4(arg1: 
->f4 : (arg1: {}) => void
+>f4 : (arg1: {    [number]: C1;}) => void

```

Comment:
same here

Reply:
The whitespace is copied from the input node (but the single-line string writer means the newlines are deleted, but the indentation is copied).